### ISSUE TITLE
Request for non-tcltk progress indicator option

### ISSUE THREAD

**Issue by ihh** [c7rishi/BAMBI#4]
Thanks for your work on this package and method.

While I appreciate that issue #2 filed by @JonathanEricson is not a bug with BAMBI and has been closed as such, a non-tcltk-dependent way of monitoring progress would be extremely helpful. I have encountered the same issue on my setup and at first I thought the method had simply hung. It was particularly confusing because an installation of BAMBI I had on earlier version of R (3.3) was hooked up to a working tcltk installation, so I was seeing the progress bar there, but that version did not include `fit_angmix` (or it didn't work) and so I had to upgrade to R 3.6.2, but tcltk doesn't seem to work on Mac with that version.

Anyway, once again, I appreciate this is not strictly BAMBI's fault but for a long-running MCMC algorithm to proceed without any visual feedback is somewhat discouraging.

**Comment by ihh** [comment]
Please do feel free to close this as a "wontfix" (or "duplicate") if you don't want to add it, I just wanted to register that it would be a v useful option for newcomers to the package.

**Comment by c7rishi** [comment]
Thanks for the feedback, and sorry for the confusion. I completely agree with your concern. Indeed,  we are also looking for a platform-independent replacement for tcltk progressbar that plays well with `future` based parallelizations.  An easy partial solution would be to just show progress on the console when chains are run in sequel/only one chain is run (we are working on this conditional case). But things get tricky when chains are run in parallel, as showing progress in future based parallel jobs is still somewhat of an unresolved problem as far as I know. Nonetheless, I will keep this issue open until at least we find a partial solution.

**Comment by ihh** [comment]
Thank you for the explanation @c7rishi !
